sgpt/cache.py

```diff
@@ -82,18 +86,32 @@ def __call__(self, func: Callable) -> Callable:
         :param func: The chat function to cache.
         :return: Wrapped function with chat caching.
         """
+
         def wrapper(*args, **kwargs):
+            # role stuff
+            role = kwargs.pop("role", None)
+            if not role:
+                role = config.get("DEFAULT_SYSTEM_ROLE")
```

Comment:
```suggestion
            role = kwargs.pop("role", None) or config.get("DEFAULT_SYSTEM_ROLE")
```

Reply:
This appears to work and is more concise! 

Want to apply this to how chat_id checks if it exists as well?